Maestro/subscriptions.json

```diff
@@ -330,7 +330,8 @@
     {
       "triggerPaths": [
         "https://github.com/dotnet/versions/blob/master/build-info/dotnet/corefx/release/uwp6.0/Latest.txt",
-        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/coreclr/release/uwp6.0/Latest.txt"
+        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/coreclr/release/uwp6.0/Latest.txt",
+        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/projectn-tfs/release/uwp6.0/Latest.txt"
```

Comment:
Looks like you're using `projectn-tfs` and not `projectn-tfs-testilc` because they both get published at around the same time--which looks good to me, just making sure.